examples/gno.land/r/demo/realm_banker/realm_banker.gno

```diff
@@ -0,0 +1,39 @@
+package realm_banker
+
+import (
+	"std"
+)
+
+var denoms map[string]std.Address = make(map[string]std.Address)
```

Comment:
top-level maps are not supported yet onchain, only in unit tests.

you should use avl.Tree for now.

Reply:
By the way, if I understand correctly, this variable allows any caller to reserve ownership of a new denom by being the first caller for that specific denom. This makes the realm suitable for testing this feature and not limited to a hardcoded admin. Am I right?

If that's the case, I really like this concept for demonstration realms. However, since it's an uncommon pattern, I suggest using more descriptive variable names and comments. Perhaps the variable could be named `denomOwners`.

Reply:
changed following
- map to avl
- minter to denomOwner (with func UpdateDenomOwner)

Reply:
This `realm_banker.gno` file was in pr for only testing. This PR isn't about demonstrating concept but to fix bugs. So I've removed old test cases and now it has txtar that only relates to denom issue.